packages/fe-fpm-writer/src/app/index.ts

```diff
@@ -3,7 +3,7 @@ import type { Editor } from 'mem-fs-editor';
 import { create } from 'mem-fs-editor';
 import { join } from 'path';
 import { lt, valid } from 'semver';
-import type { Manifest } from '@sap-ux/ui5-config';
+import type { Manifest } from '@sap-ux/project-types';
```

Comment:
With https://github.wdf.sap.corp/ux-engineering/tools-suite/pull/21282 I just try to get rid of the redundant Manifest definitions ux-spec vs. ux-specification-types. Now we have a third definition here (and a fourth, partial one at sap.fe). Which of the definitions shall be the final one? I hope, this one, as it is open source?

Reply:
@stefanschreck, perhaps there will be even one more, but I hope that will be the final one. I've contacted maintainers of https://github.com/SAP/ui5-manifest and asked to also publish the typescript types along with the JSON spec.